aspnetcore/blazor/javascript-interoperability/import-export-interop.md

```diff
@@ -326,28 +294,39 @@ export async function setMessage() {
 }
 ```
 
-`App.razor`:
+There are two options for loading the module:
 
-```razor
-@using System.Runtime.InteropServices.JavaScript
+* In `Program.cs`:
 
-<Router ...>
-    ...
-</Router>
+  ```csharp
+  using System.Runtime.InteropServices.JavaScript;
 
-@code {
-    protected override async Task OnInitializedAsync()
-    {
-        if (OperatingSystem.IsBrowser())
-        {
-            await JSHost.ImportAsync("Interop", "../js/interop.js");
-        }
-    }
-}
-```
+  ...
 
-> [!NOTE]
-> The conditional check for <xref:System.OperatingSystem.IsBrowser%2A?displayProperty=nameWithType> in the preceding example ensures that the code is only called in a Blazor WebAssembly app running on the client in a browser. If you know for sure that the code is only implemented in a Blazor WebAssembly app, you can remove the check for <xref:System.OperatingSystem.IsBrowser%2A?displayProperty=nameWithType>.
+  await JSHost.ImportAsync("Interop", "../js/interop.js");
```

Comment:
Maybe add the `IsBrowser()` check here? Your call - maybe it's already clear enough from context that this would be necessary.

Reply:
Actually, thinking about this more, I'm wondering if it's really necessary to explicitly to include the `IsBrowser()` checks in these examples at all. It would be a fairly advanced scenario to optimize a library to use these new JS interop APIs on WebAssembly only, and fall back on `IJSRuntime` when on Blazor server. I would think that in most cases, someone using these new JS interop APIs is already assuming that they're in a WASM context. We could still mention performing an `IsBrowser()` check as a way to check if you're running in a browser in case you're writing components meant to work for both Blazor Server and Blazor WebAssembly.

Just a thought - feel free to ignore it if you don't agree!

Reply:
Nevermind - it looks like you've already done this! 😄

Reply:
I agree. I just yanked them from the last example ... I'll go pull them from the code above and then just leave a NOTE about it in the text.